specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-12-01-preview/consumption.json

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
Where is this file used? This is not referenced in the readme file at all. Could you explain?

Reply:
@shalinved  Any response on this comment?

Reply:
These files shouldn't have been included in this PR in the first place. They have nothing to do with this commit. I have removed these files from this PR for now.